base/ltcmd.dtx

```diff
@@ -3300,6 +3405,266 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}{\@@_grab_f:w}
+% \changes{v1.3a}{2025-01-22}{New \texttt{f}-type grabbing function}
+% \begin{macro}{\@@_grab_f_obey_spaces:w}
+% \begin{macro}{\@@_grab_f_start:n}
+% \begin{macro}{\@@_grab_f_first:w}
+% \begin{macro}{\@@_grab_f_loop:w}
+% \begin{macro}{\@@_grab_f_auxi:w}
+% \begin{macro}{\@@_grab_f_auxii:w}
+% \begin{macro}{\@@_grab_f_auxiii:N}
+% \begin{macro}{\@@_grab_f_auxiv:}
+% \begin{macro}{\@@_grab_f_auxv:}
+% \begin{macro}{\@@_grab_f_auxvi:N}
+% \begin{macro}{\@@_grab_f_auxvii:}
+% \begin{macro}{\@@_grab_f_auxviii:}
+% \begin{macro}{\@@_grab_f_end:w}
+% \begin{macro}{\@@_grab_f_end:n}
+% \begin{macro}{\@@_grab_f_end_auxi:w}
+% \begin{macro}{\@@_grab_f_end_auxii:w}
+% \begin{macro}{\@@_grab_f_end_auxiii:w}
```

Comment:
might actually be helpful in the documented version of the envs are put closer to the actual definitions (and small groups perhaps in some cases but not like this big lumb

Reply:
I guess I favour the approach of 'keep all the `aux` functions together'. We can of course change - suggestions on a split?

Reply:
I don't mind much either way, but such long lists might actually create issues at page breaks and, you may end up with the margin note on one page but the actualy text way later, but up to you.